packages/CodingStandard/src/Fixer/Property/ArrayPropertyDefaultValueFixer.php

```diff
@@ -70,8 +70,13 @@ private function isArrayPropertyDocComment(Token $token): bool
         return true;
     }
 
-    private function addDefaultValueForArrayProperty(Token $semicolonToken): void
+    private function addDefaultValueForArrayProperty(Tokens $tokens,int $semicolonPosition): void
     {
-        $semicolonToken->setContent(' = [];');
+        $tokens->insertAt($semicolonPosition, new Token(']'));
+        $tokens->insertAt($semicolonPosition, new Token('['));
+        $tokens->insertAt($semicolonPosition, new Token(' '));
```

Comment:
@TomasVotruba , you cannot replace one token with content of multiple tokens...

Next, you cannot simply inject string, as string is pointless without context. you need to provide context-specified values, so you provide tokens.
if not, what is `class` content? what it represents? info that one is just about creating new class definition, or he is getting class name via `Foo::class` syntax ?

possible workaround:

```php
$new = Tokens::fromCode("<?php $mySuperContent=[];");
$new->clearRange(...); // drop initial code, like php tag opening

$tokens->insertAt($index, $new);
```

But... parsing the code is slow. and you are not inserting random content - you know what you want to insert.